src/ensembl/src/content/app/browser/track-panel/TrackPanel.test.tsx

```diff
@@ -41,19 +41,32 @@ describe('<TrackPanel />', () => {
     toggleTrackPanel: jest.fn()
   };
 
-  const mountBrowserImageComponent = (props?: Partial<TrackPanelProps>) =>
+  const mountTrackPanel = (props?: Partial<TrackPanelProps>) =>
     mount(<TrackPanel {...defaultProps} {...props} />);
 
   describe('rendering', () => {
     test('renders track panel when active genome is present', () => {
-      const wrapper = mountBrowserImageComponent({
+      const wrapper = mountTrackPanel({
         activeGenomeId: faker.lorem.words()
       });
       expect(wrapper.html()).not.toBe(null);
     });
 
+    test('shows track panel only if screen is or bigger than big desktop', () => {
+      const wrapper = mountTrackPanel({
+        breakpointWidth: BreakpointWidth.BIG_DESKTOP
+      });
+      expect(wrapper.props().toggleTrackPanel).toHaveBeenCalledWith(true);
+
+      jest.resetAllMocks();
+
+      wrapper.setProps({ breakpointWidth: BreakpointWidth.DESKTOP });
+      wrapper.update();
+      expect(wrapper.props().toggleTrackPanel).toHaveBeenCalledWith(false);
+    });
```

Comment:
Sorry, I am confused. I have specifically asked Andrea about our sidebar situation, and she said that sidebar should be open by default at 1200 pixels and greater. Is this test making sure that the sidebar will be closed unless screen width >= 1800? If so, then it goes against Andrea's directions.

Reply:
That's not how it is in the XD. Anyway, if Andrea says so then it probably should be opened from 1200px by default.